dev/com.ibm.ws.security.acme_fat/fat/src/com/ibm/ws/security/acme/docker/BoulderContainer.java

```diff
@@ -0,0 +1,253 @@
+            .withCommand("mysqld --bind-address=0.0.0.0 --slow-query-log --log-output=TABLE --log-queries-not-using-indexes=ON")
+            .withLogConsumer(o -> System.out.print("[SQL] " + o.getUtf8String()));
+    
+    private static final String IMAGE = "ryanesch/acme-boulder:latest";
+    
+    public BoulderContainer() {
+        super(IMAGE);
+    }
+    
+    @Override
+    protected void configure() {
+            this.withEnv("FAKE_DNS", "172.17.0.68")
+            .withEnv("PKCS11_PROXY_SOCKET", "tcp://boulder-hsm:5657")
+            .withEnv("BOULDER_CONFIG_DIR", "test/config")
+            .withEnv("GO111MODULE", "on")
+            .withEnv("GOFLAGS", "-mod=vendor")
+            .withEnv("PYTHONIOENCODING", "utf-8")
+            .withCreateContainerCmdModifier(cmd -> {
+                cmd.withDns("10.77.77.77");
+            })
+            
+            .withWorkingDirectory("/go/src/github.com/letsencrypt/boulder")
+            .withCommand("/go/src/github.com/letsencrypt/boulder/test/entrypoint.sh")
+            .withNetworkAliases("boulder")
+            .withCreateContainerCmdModifier(cmd -> cmd.withHostName("boulder"))
+    		.withExposedPorts(MANAGEMENT_PORT, LISTEN_PORT)
+            .withLogConsumer(o -> System.out.print("[BOL] " + o.getUtf8String()))
+            .withStartupTimeout(Duration.ofMinutes(3));
+    }
+    									
+    							
+    									
+
+    @Override
+    protected void containerIsCreated(String containerId) {
+        getDockerClient().connectToNetworkCmd()
+                .withNetworkId(bluenet.getId())
+                .withContainerId(containerId)
+                .withContainerNetwork(new ContainerNetwork()
+                        .withIpv4Address("10.77.77.77")
+                        .withNetworkID(bluenet.getId())
+                        .withIpamConfig(new Ipam().withIpv4Address("10.77.77.77"))
+                        .withAliases("sa1.boulder", "ca1.boulder", "ra1.boulder", "va1.boulder", "publisher1.boulder", 
+                              "ocsp-updater.boulder", "admin-revoker.boulder", "nonce1.boulder"))
+                .exec();
+        
+        getDockerClient().connectToNetworkCmd()
+                .withNetworkId(rednet.getId())
+                .withContainerId(containerId)
+                .withContainerNetwork(new ContainerNetwork()
+                        .withIpv4Address("10.88.88.88")
+                        .withNetworkID(rednet.getId())
+                        .withIpamConfig(new Ipam().withIpv4Address("10.88.88.88"))
+                        .withAliases("sa2.boulder", "ca2.boulder", "ra2.boulder", "va2.boulder", 
+                                "publisher2.boulder", "nonce2.boulder"))
+                .exec();
+    }
+    
+    @Override
+    public void start() {
+        bmysql.start();
+        bhsm.start();
+        super.start();
+    }
+    
+	public byte[] getAcmeCaIntermediateCertificate() throws Exception {
```

Comment:
No doc. Does this override a method? how is this different than the method in CAContainer?